Update to editor and plugins with updated dependencies

piemonkey opened the pull request:
## Overview
Much like the GN PR, some updates to different deps to support the latest editor and plugins.

##### connected issues and PRs:
GN equivalent: https://github.com/lblod/frontend-gelinkt-notuleren/pull/688
editor: https://github.com/lblod/ember-rdfa-editor/pull/1204
plugins: https://github.com/lblod/ember-rdfa-editor-lblod-plugins/pull/448

### Setup
N/A

### How to test/reproduce
Everything should work as before. It seems like saving with the new article nodes **does** work, but it's worth double checking (as reported by @abeforgit in Rocket Chat).

### Challenges/uncertainties
N/A

### Checks PR readiness
- [ ] UI: works on smaller screen sizes
- [ ] UI: feedback for any loading/error states
- [ ] Check cancel/go-back flows
- [ ] Check database state correct when deleting/updating (especially regarding relationships)
- [x] changelog
- [x] npm lint
- [ ] no new deprecations

elpoelma reviewed (changes requested):
Looks good overall, some issues:
- We still have the option enabled to insert a snippet, but the `snippet` node has not been added to the schema. IMO it does not really make sense to allow template creators to already insert snippets (only snippet placeholders)

piemonkey commented:
> Looks good overall, some issues:
> 
>     * We still have the option enabled to insert a snippet, but the `snippet` node has not been added to the schema. IMO it does not really make sense to allow template creators to already insert snippets (only snippet placeholders)

Oh, forgot to mention, I removed the snippet insert button and updated to the new patch release of the plugins.

elpoelma reviewed (approved):
Looks good now!
